_posts/2025-01-30-socratic-seminar-31.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+type: socratic
+title: "Seminário Socrático 031"
+meetup: https://www.meetup.com/clube-bitcoin-floripa/events/304755077/
+---
+
+## Avisos
+
+- Entrem no grupo do Whatsapp [São Paulo Bitdevs](https://chat.whatsapp.com/HiaPqjmUqER5djFPR1Yl3T)!
+- Respeite a privacidade dos participantes.
+- Os meetups nunca são gravados. Queremos todos a vontade para participar e discutir os assuntos programados, de forma anônima se assim o desejarem.
+
+## Agradecimentos
+
+- Agradecemos à Vinteum pela casa, comidas e bebidas.
+
+## Cronograma
+
+### Warmup
+
+* [A Cypherpunk's Manifesto](https://www.activism.net/cypherpunk/manifesto.html)
+* [Donald Trump Signs 'A Full and Unconditional Pardon' for Silk Road Founder Ross Ulbricht](https://www.nobsbitcoin.com/donald-trump-frees-ross/)
+* [Court lifts OFAC sanctions on Tornado Cash, marking a major win for crypto privacy advocates.](https://xcancel.com/cointelegraph/status/1882001200524173553?s=46&t=757GlHmx5xzJQmvWmVK2Sg)
+     - [Court Order](https://storage.courtlistener.com/recap/gov.uscourts.txwd.1211705/gov.uscourts.txwd.1211705.99.0.pdf)
+* Tabconf 2024 and OP_NEXT Recordings
+    - [Tabconf](https://www.youtube.com/@tabconf/videos)
+    - [OP_NEXT](https://blockspace.media/opnext-bitcoin-scaling/)
+* [CTB6 codebase](https://github.com/CasaVinteUm/ctb6)
+* [Mempool bugado](https://x.com/0xB10C/status/1869781703646146706)
```

Comment:
I think I suggested this one but it is kinda "well, whatever".

We can remove this one